js/apps/account-ui/src/root/Root.tsx

```diff
@@ -1,13 +1,16 @@
-import { KeycloakProvider } from "@keycloak/keycloak-ui-shared";
+import {
+  initializeDarkMode,
+  KeycloakProvider,
+} from "@keycloak/keycloak-ui-shared";
 import { Page, Spinner } from "@patternfly/react-core";
-import { Suspense } from "react";
+import { Suspense, useEffect } from "react";
 import { Outlet } from "react-router-dom";
-
 import { environment } from "../environment";
 import { Header } from "./Header";
 import { PageNav } from "./PageNav";
 
 export const Root = () => {
+  useEffect(() => initializeDarkMode(), []);
```

Comment:
Since there is no dependency between this code and React I would suggest to put this on top in `main.tsx` so that it runs pretty much immediately (e.g. doesn't wait for i18n, etc.).